client/a8c-for-agencies/sections/overview/sidebar/contact-support/style.scss

```diff
@@ -0,0 +1,24 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.overview__contact-support-button {
+	width: 100%;
+}
+
+// Zendesk Chat Button
+iframe#launcher {
+	right: 24px !important;
+
+	@include break-wide() {
+		bottom: 24px !important;
+	}
+}
+
+// Zendesk Chat Window
+iframe[title="Messaging window"] {
+	margin-right: 12px;
+
+	@include break-wide() {
+		margin-bottom: 8px;
+	}
+}
```

Comment:
[Non-blocker] It is good practice to use [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) since we can technically support RTL locales in Calypso.

Reply:
May not be possible here as the positioning is determined by the widget settings in Zendesk admin (available options are just "left" or "right") 🤔